app/views/foreman_ansible/job_templates/capsule_upgrade_-_ansible_default.erb

```diff
@@ -71,3 +74,33 @@ feature: ansible_run_capsule_upgrade
         - name: satellite-maintain upgrade return code is non-zero
           fail:
             msg: "Failed! Capsule server upgrade failed. See /var/log/foreman-installer/capsule.log in the Capsule server for more information"
+<% else -%>
+- hosts: all
+  vars:
+    foreman_release_version: "<%= input('target_version').present? ? input('target_version') : foreman_short_version %>"
+  tasks:
+    - name: Gather the rpm package facts
+      package_facts:
+        manager: auto
+
+    - name: Fail if the target server is a Foreman server
+      fail:
+        msg: "This playbook cannot be executed on a Foreman server. Use only on a Smart Proxy server."
+      when: "'foreman' in ansible_facts.packages"
+
+    - name: Update foreman repositories
+      package:
+        name: https://yum.theforeman.org/releases/{{ foreman_release_version }}/el{{ ansible_distribution_major_version }}/{{ ansible_architecture }}/foreman-release.rpm
```

Comment:
On RHEL 8 this is failing on:
```
TASK [Update foreman repositories] *********************************************
  21:
fatal: [proxy33.example.com]: FAILED! => {"changed": false, "msg": "Failed to validate GPG signature for foreman-release-3.4.0-1.el8.noarch: Public key for foreman-release3o_lm6dt.rpm is not installed"}
  22:
PLAY RECAP ***
```

However if I run command manually it works without any problems:
```bash
dnf -y install https://yum.theforeman.org/releases/3.4/el8/x86_64/foreman-release.rpm
```

Little rewrite of playbook helped, but I guess it's not the way we wan't to go:
```yaml
    - name: Update foreman repositories
      yum:
        name: https://yum.theforeman.org/releases/{{ foreman_release_version }}/el{{ ansible_distribution_major_version }}/{{ ansible_architecture }}/foreman-release.rpm
        disable_gpg_check: yes
        state: present
```